ui/src/app/pages/access/access-renewal.tsx

```diff
@@ -195,18 +203,12 @@ export const AccessRenewal = fp.flow(withProfileErrorModal)((spinnerProps: WithS
 
   const expirableModules = modules.filter(moduleStatus => accessRenewalModules.includes(moduleStatus.moduleName));
 
-  const completeOrBypassed = moduleName => {
-    const status = modules.find(m => m.moduleName === moduleName);
-    const wasBypassed = !!status.bypassEpochMillis;
-    return wasBypassed || !isExpiring(status.expirationEpochMillis);
-  }
-
-  const allModulesCompleteOrBypassed = fp.flow(fp.map('moduleName'), fp.all(completeOrBypassed))(expirableModules);
+  const renewalCompleteAndNotExpiring = expirableModules.every(bypassedOrCompleteAndNotExpiring);
 
   // onMount - as we move between pages, let's make sure we have the latest profile and external module information
   useEffect(() => {
     const expiringModules = expirableModules
-      .filter(status => status.expirationEpochMillis && isExpiring(status.expirationEpochMillis))
```

Comment:
not actually a change in behavior; the old `isExpiring()` ignored the undefined case, and the new `isExpiring()` returns false for undefined.